posts/2021-06-04-database-ssl-primer.adoc

```diff
@@ -222,6 +229,8 @@ To establish trust, configure one of the following options:
 
 > NOTE: the `ewallet.p12` file contains additional encrypted data that the Oracle driver needs to establish the TLS connection to the database.  As such, the default Security Providers shipped with the Java JRE will fail to read this file. If using the `ewallet.p12` keystore users will need to add the Oracle PKI driver to their JVM native liberty path, and add the `oracle.security.pki.OraclePKIProvider.OraclePKIProvider` class to position 1 of their security provides.
 
+> NOTE: prior to Oracle JDBC 23 the Oracle PKI driver (oraclepki.jar) depended on osdt_core.jar and osdt_cert.jar for Oracle Wallet support. After Oracle JDBC 23 the Oracle PKI driver includes these dependencies and are no longer required.
```

Comment:
```suggestion
> NOTE: prior to Oracle JDBC 23 the Oracle PKI driver (`oraclepki.jar`) depended on the `osdt_core.jar` and `osdt_cert.jar` files for Oracle Wallet support. After Oracle JDBC 23, the Oracle PKI driver includes these dependencies and the files are no longer required.
```

Reply:
not sure ive got the end part right, but as written it seemed a little confusing- if they are no longer required, why does the driver include them?